action/login/prompt.go

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) 2020 Target Brands, Inc. All rights reserved.
+//
+// Use of this source code is governed by the LICENSE file in this repository.
+
+package login
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/manifoldco/promptui"
+
+	"github.com/sirupsen/logrus"
+)
+
+// PromptUsername asks the user to provide a username via
+// terminal input based off the provided configuration.
+func (c *Config) PromptUsername(in io.ReadCloser) error {
+	logrus.Debug("executing prompt for username for login configuration")
+
+	// create variable to store errors
+	var err error
+
+	// create the prompt for a username
+	p := promptui.Prompt{
+		Label: "Please enter a username: ",
+		Stdin: in,
+	}
+
+	// run the prompt to capture the username from the input
+	c.Username, err = p.Run()
+	if err != nil {
+		return err
+	}
+
+	logrus.Trace("checking username input provided")
+
+	// check if login username is set
+	if len(c.Username) == 0 {
+		return fmt.Errorf("no login username provided")
+	}
+
+	return nil
+}
+
+// PromptPassword asks the user to provide a password via
+// terminal input based off the provided configuration.
+func (c *Config) PromptPassword(in io.ReadCloser) error {
```

Comment:
This is where we add the new `PromptPassword()` function.

The purpose of this function is to create a prompt for the password via the terminal session.